src/PerfTestHarness/Program.cs

```diff
@@ -12,7 +13,7 @@ private static void Main(string[] args)
         {
             var iterations = Enumerable.Range(1, 500000);
             var client = new StatsDUdpTransport(10, "localhost", 3128);
-            var formatter = new StatsDMessageFormatter();
+            var formatter = new StatsDMessageFormatter(CultureInfo.InvariantCulture);
```

Comment:
IDK why it's like that. I'm happy to change to use `InvariantCulture` as the default.